### DISCUSSION TITLE
How to change the "report" schedule?

### DISCUSSION THREAD

**Discussion by thefunkygibbon** [netalertx/NetAlertX#898] [General]
I've no idea what i've done but i seem to be getting reports generated every 5-10 mins,, which I don't really look at anyway, so i always get a "alert" in the alarm bell thing in the corner with crazy large numbers in it every time i log in.   is there  a way of scheduling these reports to be once a day or something?

**Comment by jokob-sk** [comment]
Hi, 

depends on what's going on - if you changed settings, you can compare your setting backups in the /config folder from before this started occurring.

**Comment by thefunkygibbon** [comment]
i don't really know.  its always been like this

![image](https://github.com/user-attachments/assets/d087c741-3cc1-4801-978c-776faa7572d6)
![image](https://github.com/user-attachments/assets/e844f627-2c2e-45a5-a902-12d4442a0e9c)
and thats just in the last 10 hours

is there a setting for when reports are sent?  or is a report generated after each time any network scan etc takes place (this is what it feels like)?  if so is there any way of stopping/changing that?

**Reply by jokob-sk** [reply]
You can find a detailed description on how notifications work here: https://github.com/jokob-sk/NetAlertX/blob/main/docs/NOTIFICATIONS.md
